README.md

```diff
@@ -30,24 +30,35 @@ Complete configuration for the different [node.js versions](https://github.com/n
 # https://github.com/nodejs/LTS
 sudo: false
 language: node_js
+os:
+ - linux
```

Comment:
Why is this needed?

Reply:
it's not - it's implicit - but it leaves a convenient location for adding OS X or Windows if needed in the future (plus, explicit > implicit)